source/common/ipc-api.js

```diff
@@ -33,3 +33,9 @@ export const DOWNLOAD_LOGS = {
   SUCCESS: `${DOWNLOAD_LOGS_CHANNEL}-success`,
   ERROR: `${DOWNLOAD_LOGS_CHANNEL}-error`,
 };
+const UPDATE_API_CHANNEL = 'update-api';
+const CLIENT_REQ_API_CHANNEL = 'client-req-api';
+export const UPDATE_API = {
```

Comment:
@cleverca22, this is called `UPDATE_API`, which could lead one to think it's Update-System-related -- which doesn't seem to be true, if I'm not terribly wrong.

Somewhat similar is the `CLIENT_REQ` name.

AFAICT, this is related to "communication setup", in the sense that exchanged information includes the TLS credentials.  Maybe we could change the names to make this more clear -- again, unless I'm misinterpreting things?

Reply:
yeah, these are used for IPC between the main and render process, within daedalus, to forward the data it got from cardano's IPC channel
they could be renamed a bit if better names can be found